Sources/OpenAI/Supporting Types/Safety.swift

```diff
@@ -0,0 +1,61 @@
+/**
+ The result of a contentFilter request.
+ */
+public enum Safety: Int {
+    
```

Comment:
Less cumbersome to adopt `Hashable` (and therefore `Equatable`) in the declaration.

```suggestion
public enum Safety: Int, Hashable {
```